cuchem/requirements.txt

```diff
@@ -25,4 +25,5 @@ plotly==4.9.0
 pytest==6.2.2
 umap-learn==0.5.1
 grpcio
+biopython
```

Comment:
@rilango I think we don't need biopython anymore since there's no processing left to do. I will approve though -- will test this soon.

Reply:
yeah.. I will do the cleanup and merge.